cinn/ir/ir_schedule.h

```diff
@@ -216,6 +216,13 @@ class IRSchedule {
    */
   Expr CacheWrite(const Expr& block, int write_buffer_index, const std::string& memory_type);
 
+  /**
+   * \brief Add SyncThreads statements in AST.
+   * @param ir_node The insertion point in AST.
+   * @param after_node Whether to insert the statement after the insertion point.
```

Comment:
So, we will insert the statement before the insertion point when it is False? Suggest to explain the two cases.

Reply:
Done, thanks